src/prpy/planning/ompl.py

```diff
@@ -163,10 +164,40 @@ def _Plan(self, robot, goal=None, tsrchains=None, timeout=30., shortcut_timeout=
         return traj
 
 
-class RRTConnect(OMPLPlanner):
-    def __init__(self):
-        OMPLPlanner.__init__(self, algorithm='RRTConnect')
+class OMPLRangedPlanner(OMPLPlanner):
+    """ Construct an OMPL planner with a default 'range' parameter set.
+
+    This class wraps OMPLPlanner by setting the default 'range' parameter,
+    which defines the length of an extension in algorithms like RRT-Connect, to
+    include an integer number 'multipler' of state validity checks.
+
+    @param algorithm name of the OMPL planner to create
+    @param robot_checker_factory robot collision checker factory
+    @param multiplier number of state validity checks per extension
+    """
+    def __init__(self, algorithm='RRTConnect', robot_checker_factory=None,
+            multiplier=1):
```

Comment:
I'm actually a bit wary of the default being 1 here, since that doesn't tend to give good performance.  (Did @gilwoolee do experiments on this for ISER?)  The OMPL default is 20% of the space extent.  Perhaps we could default to something similar?  I'm not sure what the right thing to do is here, but I would hate to see performance suffer as a result of this default.  Or maybe the old code was also effectively defaulting to a unit multiplier?